ee/dtr/admin/install/index.md

```diff
@@ -26,14 +26,14 @@ Before installing DTR, confirm that your infrastructure meets DTR's [system requ
 
 ## 2. Install UCP
 
-To run, DTR requires Docker Enterprise's Docker Universal Control Panel (UCP). If UCP is not yet installed, refer to [install UCP for production](/ee/ucp/admin/install/).
+To run, DTR requires Docker Enterprise's UCP. If UCP is not yet installed, refer to [install UCP for production](/ee/ucp/admin/install/).
```

Comment:
or how about "Universal Control Plane must be installed to run DTR." I would avoid "Docker Enterprise's UCP".

Reply:
Now reads "UCP must be installed to run DTR." (The acronym is defined at the top of the topic.)